File: UI/AlAmeen/src/app/models/CustomerAddress.ts
import { Customer } from "./Customer";

export interface CustomerAddress {
    customerAddressId: number;
    addressLine1: string;
    addressLine2: string;
    country: string;
    city: string;
    state: string;
    postalCode: string;
    isShippingAddress: boolean;
    isBillingAddress: boolean;
    customerId: number;
    customer: Customer;
}

File: API/AlAmeen/Domain/Entites/CustomerAddress.cs
namespace Domain.Entites
{
    public class CustomerAddress
    {
        public int CustomerAddressId { get; set; }
        public string AddressLine1 { get; set; }
        public string AddressLine2 { get; set; }
        public string Country { get; set; }
        public string City { get; set; }
        public string State { get; set; }
        public string PostalCode { get; set; }
        public bool IsShippingAddress { get; set; }
        public bool IsBillingAddress { get; set; }

        public int CustomerId { get; set; }
        public Customer? Customer { get; set; }

    }
}
